### PR TITLE
WL-4394 Change Markbook to Gradebook.

### PR DESCRIPTION
**Pull request by RebeccaMiller-Which**

### CODE REVIEW

File: gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties [hunk 1]
```diff
@@ -1,210 +1,126 @@
...
-import_assignment_header=Markbook Item
 grader_rules_no_rules=There are currently no rules defined for this marker.
 export_grade_for_sis_prefix=course_mark_sis
-add_assignment_page_title=Add Markbook Item(s)
-edit_assignment_instruction=Submit this form to edit a markbook item in the markbook.
-remove_assignment_success=Markbook item {0} has been deleted.
-student_view_assignments=Markbook Items
-loading_dock_instructions=Below is the loading dock, a holding place for spreadsheets (csv format) that have been uploaded from your local computer. <br> Once loaded here, the data in the spreadsheet can be imported in part or whole into your markbook.
-assignment_details_page_title=Markbook Item Summary
+loading_dock_instructions=Below is the loading dock, a holding place for spreadsheets (csv format) that have been uploaded from your local computer. <br> Once loaded here, the data in the spreadsheet can be imported in part or whole into your gradebook.
```
Comment (buckett) [inline gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties:91]: This line can go I think as it doesn't reference marks.

File: gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties [hunk 2]
```diff
@@ -1,210 +1,126 @@
...
 gb_setup_no_grade_entry_scale=No mark entry scale for letter marking has been defined.
-import_entire_some_success=Some markbook items were imported successfully. See below for the errors that did occur.  
 feedback_options_cannot_change_percentage2=Campus Teaching Centre
-feedback_options_cannot_change_percentage1=Alert\: Due to problems with this markbook configuration your changes have not been saved. Please contact your 
-roster_export_csv=Export Markbook
+roster_export_csv=Export Gradebook
```
Comment (buckett) [inline gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties:94]: This line can go I think as it doesn't reference marks.

File: lessonbuilder/tool/src/resources/messages_en_GB.properties [hunk 3]
```diff
@@ -1,9 +1,6 @@
 # Gradebook
-simplepage.comments-graded=Create Markbook entries for these
-simplepage.gbname-expected=Markbook item name must be supplied.
-simplepage.no-gradebook=This item couldn't be marked for grading. The most likely issue is that no Markbook tool has been added to this site yet.
-simplepage.page.gradebook=Create Markbook entry when page is completed.
-simplepage.question-gradebook-title=Title in Markbook:
+simplepage.comments-graded=Create Gradebook entries for these
```
Comment (buckett) [inline lessonbuilder/tool/src/resources/messages_en_GB.properties:2]: This line can go I think as it doesn't reference marks.

File: lessonbuilder/tool/src/resources/messages_en_GB.properties [hunk 4]
```diff
@@ -1,9 +1,6 @@
...
+simplepage.page.gradebook=Create Gradebook entry when page is completed.
```
Comment (buckett) [inline lessonbuilder/tool/src/resources/messages_en_GB.properties:3]: This line can go I think as it doesn't reference marks.

**Review by buckett — DISMISSED** [review]
A few lines seem to remain even though they don't reference marks, is this because they are different?

File: gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties (hunk 1 above)
Comment (RebeccaMiller-Which) [reply gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties:91]: The default properties file refers to csv or xls format so I changed the en_GB entry to refer to gradebook.

File: gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties (hunk 2 above)
Comment (RebeccaMiller-Which) [reply gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties:94]: The default property is 'Export as CSV' so I made the change above.

File: lessonbuilder/tool/src/resources/messages_en_GB.properties (hunk 3 above)
Comment (RebeccaMiller-Which) [reply lessonbuilder/tool/src/resources/messages_en_GB.properties:2]: The default entry is 'Create Gradebook items for these' so I made the change above.

File: lessonbuilder/tool/src/resources/messages_en_GB.properties (hunk 4 above)
Comment (RebeccaMiller-Which) [reply lessonbuilder/tool/src/resources/messages_en_GB.properties:3]: The default entry is 'Create Gradebook item when page is completed.' so I made the change above.

File: gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties (hunk 1 above)
Comment (buckett) [reply gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties:91]: Cool.

File: gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties (hunk 2 above)
Comment (buckett) [reply gradebook/app/ui/src/bundle/org/sakaiproject/tool/gradebook/bundle/Messages_en_GB.properties:94]: Ok.

File: lessonbuilder/tool/src/resources/messages_en_GB.properties (hunk 3 above)
Comment (buckett) [reply lessonbuilder/tool/src/resources/messages_en_GB.properties:2]: Thanks.

File: lessonbuilder/tool/src/resources/messages_en_GB.properties (hunk 4 above)
Comment (buckett) [reply lessonbuilder/tool/src/resources/messages_en_GB.properties:3]: Ok

**Pull request merged**